retrieval/client/fsm_test.go

```diff
@@ -42,7 +42,7 @@ type mockClientEnvironment struct {
 }
 
 func (e *mockClientEnvironment) OpenDataTransfer(ctx context.Context, to peer.ID, proposal *deal.Proposal) (datatransfer.ChannelID, error) {
-	return datatransfer.ChannelID{ID: datatransfer.TransferID(rand.Uint64()), Responder: to, Initiator: testnet.GeneratePeers(1)[0]}, e.OpenDataTransferError
+	return datatransfer.ChannelID{ID: datatransfer.TransferID(rand.Uint64()), Responder: to, Initiator: GeneratePeers(1)[0]}, e.OpenDataTransferError
```

Comment:
If GeneratePeers is only used here I would literally put: `peer.ID(1)` there and not make the method.